site/snippets/header.php

```diff
@@ -18,5 +18,61 @@
   <?php echo js('js/libs/modernizr.js') ?>
   <?php echo js('js/main.js') ?>
 
+
+<!-- jQuery for the Contact Form-->
```

Comment:
Yeah we should never put jquery code in here if we can avoid it :) just for future reference. js/main.js is where all the javascript goes.